### PR TITLE
Update maintainers info and copyright for year 2019

### PR DESCRIPTION
**Pull request by ejeschke**
Update maintainers info and copyright for year 2019.  

Also fixes a broken load of banner using Menu->About.

### CODE REVIEW

**Comment by ejeschke** [comment]
@pllim, these changes ok with you?

File: ginga/util/loader.py [hunk 1]
```diff
@@ -118,5 +118,6 @@ def add_loader(mimetype, loader):
     add_loader(mimetype, load_asdf)
 
 # ### RGB ###
-for mimetype in ['image/jpeg', 'image/png', 'image/tiff', 'image/gif']:
+for mimetype in ['image/jpeg', 'image/png', 'image/tiff', 'image/gif',
+                 'image/ppm', 'image/pnm', 'image/pbm', 'image/pnm']:
```
Comment (pllim) [inline ginga/util/loader.py]: `pnm` was listed twice. Did you mean something else?
```suggestion
                 'image/ppm', 'image/pnm', 'image/pbm']:
```

**Comment by pllim** [comment]
p.s. What is `icons/ginga.fits` used for?

File: ginga/util/loader.py (hunk 1 above)
Comment (hcferguson) [reply ginga/util/loader.py]: I think it's fine to include a mention of STScI there. Thanks!

Comment (ejeschke) [reply ginga/util/loader.py]: Good catch, @pllim.  I think that is in there in error.  I'll update the PR.

**Comment by ejeschke** [comment]
> p.s. What is icons/ginga.fits used for?

That's a good question!  It looks like it was committed quite a while ago.  My guess is that at some point I thought it would be nice to include one fits file with the package for testing. If we get around to adding more test files, we can properly stick it in a directory or get rid of it.

**Pull request merged**